fix: Skip download and installation saucectl on Sauce (#31)
saucelabs/node-saucectl: README.md

(build 7468a24c788b4ca4d67d50372c839edf03e5df6a)
```

__Note:__ When you run the command for the first time, it will initially download the binary. This only happens once.

> __Note:__ `saucectl` installation is disabled on Sauce Labs Cloud. If you wish to force the installation, set the `FORCE_INSTALL_SAUCECTL` environment variable to `true`.

Reviewer: Maybe `SAUCECTL_FORCE_INSTALL` instead to match the pattern of the other env vars (e.g. `SAUCECTL_INSTALL_BINARY`)

### Install from a Specified Source

> If you need to download `saucectl` from a specific source, or if you are using `npx saucectl`, set the following environment variable:

Author: > or if you are using `npx saucectl`

Am I getting it correct? I'm confused.

> If you need to download `saucectl` from a specific source, or if you are using `npx saucectl`, set the following environment variable:

Reviewer 2: How about this:
```suggestion
If you want the installer to download `saucectl` from a specific source, set the following environment variable:
```
